modules/recommended-scale-practices.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/recommended-cluster-scaling-practices.adoc
+
+[id="recommended-scale-practices_{context}"]
+= Recommended practices for scaling the cluster
+
+When scaling up the cluster to higher node counts, spread the nodes across all
+of the available zones for higher availability. The controller might not be able
+to create the machines if the replicas in the MachineSets are set to higher
```

Comment:
This section might be better as a bulleted list.  I don't think users really need too much detail on the specifics as to why this is the case (maybe add a short note after the list)

- When scaling up the cluster, spread nodes across all of the available zones for higher availability.
- When scaling up the cluster, it is recommended to scale up by no more than 25 to 50 machines at once.
- Consider creating new MachineSets in each available zone with alternative instance types of similar size (e.g. On AWS, m5.large and m5d.large) to help mitigate any periodic provider capacity constraints.

Note:  Cloud providers may implement a quota for API services, thus a gradual scaling of the cluster is recommended.